test/Libraries/RevitIntegrationTests/GlobalParameterTests.cs

```diff
@@ -17,7 +17,7 @@ namespace RevitSystemTests
     class GlobalParameterTests : RevitSystemTestBase
     {
         [Test]
-        [TestModel(@".\empty-2017.rvt")]
```

Comment:
@sharadkjaiswal Yeah, but this file is not there in this branch. Also empty.rvt in this branch is from Revit 2016 and does not take long. I tested locally the test passed with this change